rufs/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "rufs"
+version = "0.3.0"
+edition = "2021"
+license = "BSD-2-Clause"
+authors = ["Benjamin Stürz <benni@stuerz.xyz>", "Alan Somers <asomers@gmail.com>", "Davids Paskevics <davids.paskevics@gmail.com>"]
+description = "FUSE implementation of FreeBSD's UFSv2"
+repository = "https://github.com/realchonk/fuse-ufs"
+rust-version = "1.74.0"
+documentation = "https://docs.rs/rufs"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+fuser = ["dep:fuser"]
```

Comment:
Is fuser even used in rufs, or is it only used in fuse-ufs ?

Reply:
https://github.com/realchonk/fuse-ufs/pull/74/files#diff-30f68d7e587919ed03125a77eacd3add233695dead48aec6bf8d060b75b64deeR173